colab/demo.ipynb

```diff
@@ -58,59 +61,57 @@
         "* **WARNING: Running pip as the 'root' user...:**:\n",
         "    * This and other pip warnings or errors can be safely ignored.\n",
         "* **Other problems?**\n",
-        "    * Feel free to create an issue on our [GitHub repo](https://github.com/nerfstudio-project/nerfstudio).\n"
+        "    * Feel free to create an issue on our [GitHub repo](https://github.com/nerfstudio-project/nerfstudio)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Nerfstudio Demo"
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "9oyLHl8QfYwP",
-        "cellView": "form"
+        "cellView": "form",
+        "id": "9oyLHl8QfYwP"
       },
       "outputs": [],
       "source": [
-        "#@markdown <h1>Install Nerfstudio and Dependencies (~8 min)</h1>\n",
```

Comment:
Does this still take around 8min?

Reply:
Will check, it took kind of long for me.